components/TokenView/src/TokenView/TokenViewOrientation.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace CommunityToolkit.Labs.WinUI;
+
+public enum TokenViewOrientation
```

Comment:
Separated out this enum as otherwise it's in a weird namespace.

However, I'm not a big fan of this property... Do we see more than two options here? Either we could have a boolean like `IsWrapped` or call it `WrappingMode` instead with like `NoWrap` and `Wrap` (like `TextWrapping` on `RichTextBlock`)?